### ISSUE TITLE
Default CSVFILE does not work for GENDATA_RFT

### ISSUE THREAD

**Issue by berland** [equinor/semeio#147]
The default filename for CSV export from GENDATA_RFT apparently does not work:

```
/scratch/fmu/havb/py3_r001_reek/realization-0/iter-0]$ grep CSVFILE jobs.json 
  "argList" : ["-e","eclipse/model/PY3_R001_REEK-0","-t","/project/fmu/public/reek/resmod/ff/2019a/r001/ert/model/../input/observations/rft/","-w","/project/fmu/public/reek/resmod/ff/2019a/r001/ert/model/../input/observations/rft/well_date_rft.txt","-z","/project/fmu/public/reek/resmod/ff/2019a/r001/ert/model/../input/observations/rft/layer_zone_table.txt","-c","<CSVFILE>"],
```

where `<CSVFILE>` above should be `gendata_rft.csv` according to the job configuration.

**Comment by berland** [comment]
Solved by #148

**Comment by berland** [comment]
(CI fails due to `black` bumped to 20.8b0 on CI nodes)
